packages/mattermost-redux/src/reducers/entities/threads/counts.ts

```diff
@@ -59,23 +63,24 @@ function handleReadChangedThread(state: ThreadsState['counts'], action: GenericA
     const counts = state[teamId] ? {
         ...state[teamId],
     } : {
-        total_unread_threads: 0,
-        total: 0,
-        total_unread_mentions: 0,
+        total_unread_threads: prevUnreadReplies,
+        total: 1,
```

Comment:
Is changing the default total number of threads here intentional?

Reply:
nope, thanks!